libraries/stdlib/samples/test/samples/collections/collections.kt

```diff
@@ -799,4 +799,17 @@ class Collections {
         }
 
     }
+
+    class Filtering {
+        @Sample
+        fun filterNotNull() {
+            val numbers: List<Int?> = listOf(1, 2, null, 4)
+            val nonNullNumbers = numbers.filterNotNull()
+
+            assertEquals(
```

Comment:
Better to use `assertPrints` and keep it in a single line. Then it will be replaced with `println(...)` in the runnable sample.

Reply:
Thanks!